examples/2016-10-31/lambda_edge/README.md

```diff
@@ -9,7 +9,7 @@ LambdaEdgeFunctionSample:
     Type: AWS::Serverless::Function
     Properties:
         CodeUri: src/
-        Runtime: nodejs6.10
+        Runtime: nodejs12.x
```

Comment:
Following the [lambda@edge requirements guide](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-requirements-limits.html) node12 isn't supported!

Reply:
@timoschilling thank you for checking this. I will update the PR.